iidm-api/src/main/java/eu/itesla_project/iidm/network/util/Networks.java

```diff
@@ -298,52 +349,64 @@ public static boolean checkFlows(String id, double r, double x, double rho1, dou
         double p2_calc = rho2 * rho1 * u2 * u1 * y * Math.sin(theta2 - theta1 - ksi + alpha2 - alpha1) + rho2 * rho2 * u2 * u2 * (y * Math.sin(ksi) + g2);
         double q2_calc = - rho2 * rho1 * u2 * u1 * y * Math.cos(theta2 - theta1 - ksi + alpha2 - alpha1) + rho2 * rho2 * u2 * u2 * (y * Math.cos(ksi) - b2);
 
-        boolean debug = false;
-        if (debug) {
-            System.out.println("r=" + r);
-            System.out.println("x=" + x);
-            System.out.println("g1=" + g1);
-            System.out.println("g2=" + g2);
-            System.out.println("b1=" + b1);
-            System.out.println("b2=" + b2);
-            System.out.println("rho1=" + rho1);
-            System.out.println("rho2=" + rho2);
-            System.out.println("alpha1=" + alpha1);
-            System.out.println("alpha2=" + alpha2);
-            System.out.println("u1=" + u1);
-            System.out.println("u2=" + u2);
-            System.out.println("theta1=" + theta1);
-            System.out.println("theta2=" + theta2);
-            System.out.println("z=" + z);
-            System.out.println("y=" + y);
-            System.out.println("ksi=" + ksi);
-            System.out.println("p1=" + p1);
-            System.out.println("q1=" + q1);
-            System.out.println("p2=" + p2);
-            System.out.println("q2=" + q2);
+        formatter.writeCell(id).writeCell("expected_p1").writeCell(p1)
+                 .writeCell(id).writeCell("calculated_p1").writeCell(p1_calc)
```

Comment:
I think the expected and calculated p1 are the same (theoretical value). We can rename "expected" by "network" and "calculated" by "expected".